code/game/objects/structures/dropship_equipment.dm

```diff
@@ -362,6 +368,8 @@
 /obj/structure/dropship_equipment/sentry_holder/proc/deploy_sentry()
 	if(!deployed_turret)
 		return
+	setDir(ship_base.dir)
+	deployed_turret.setDir(dir)
```

Comment:
Won this also rotate turrets facing to the side(like on the alamo)?

Reply:
ship_base is actually the weapon attach mount which has its own dir thats different from the direction of the shuttle. 

I had to add this because the dir of turrets didn't get properly updated when the ship rotated.
The direction was only calculated when the powerloader attached it on the mount.

So the Alamo turrets will be fine.